.github/workflows/test.yml

```diff
@@ -3,6 +3,20 @@ on: [push, pull_request]
 name: Test
```

Comment:
Please add a test for `cargo test --release --no-default-features`

The doc test in `range_proof.rs` can be excluded from this test if appended like this:
```
/// #[cfg(feature = "rand")]
/// use rand::rngs::OsRng;
/// # fn main() {
/// #[cfg(feature = "rand")]
/// {
/// ...
```